cgmes/cgmes-extensions/src/main/java/com/powsybl/cgmes/extensions/CgmesMetadataModelsSerDe.java

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.extensions;
+
+import com.google.auto.service.AutoService;
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.extensions.AbstractExtensionSerDe;
+import com.powsybl.commons.extensions.ExtensionSerDe;
+import com.powsybl.commons.io.DeserializerContext;
+import com.powsybl.commons.io.SerializerContext;
+import com.powsybl.commons.io.TreeDataReader;
+import com.powsybl.commons.io.TreeDataWriter;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.serde.NetworkSerializerContext;
+
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.Map;
+
+/**
+ * @author Luma Zamarreño {@literal <zamarrenolm at aia.es>}
+ */
+@AutoService(ExtensionSerDe.class)
+public class CgmesMetadataModelsSerDe extends AbstractExtensionSerDe<Network, CgmesMetadataModels> {
+
+    private static final String MODEL = "Model";
+    private static final String PROFILE = "Profile";
+    private static final String DEPENDENT_ON = "DependentOn";
+    private static final String SUPERSEDES = "Supersedes";
+    private static final String MODELS = "Models";
+    private static final String PROFILES = "Profiles";
+    private static final String DEPENDENT_ONS = "DependentOns";
+    private static final String SUPERSEDESS = "Supersedess";
+
+    public CgmesMetadataModelsSerDe() {
+        super(CgmesMetadataModels.NAME, "network", CgmesMetadataModels.class,
+                "cgmesMetadataModels.xsd",
+                "http://www.powsybl.org/schema/iidm/ext/cgmes_metadata_models/1_0", "cmm");
+    }
+
+    @Override
+    public Map<String, String> getArrayNameToSingleNameMap() {
+        return Map.of(
+                MODELS, MODEL,
+                PROFILES, PROFILE,
+                DEPENDENT_ONS, DEPENDENT_ON,
+                SUPERSEDESS, SUPERSEDES
+                );
```

Comment:
Just to recap, this map is used to translate the tag which might occur more than once in xiidm within the same tag to a list/plural tag in jiidm, as we cannot put the same attribute name twice in json. In this sense, `DependentOns` and Superserdess` are not correct.

Even if it's merely a matter of grammar (hence no reason to be different from one usage to the other) we should protect this by associating it with a namespace. Nonetheless this is not straightforward, so we need to put something else while waiting for it, for instance:
- DependentOnModel / DependentOnModels
- SupersedesModel / SupersedesModels
If we change it later, this means we will not be able to read the jiidm files produced previously. We will then deal with it by adding a 2.0 version to this SerDe.

Reply:
The names have been fixed following your proposal